examples/jmespath_functions/src/powertools_base64_jmespath_function.py

```diff
@@ -0,0 +1,7 @@
+import powertools_base64_jmespath_schema as schemas
+
+from aws_lambda_powertools.utilities.validation import validate
+
+sample_event = {"data": "eyJtZXNzYWdlIjogImhlbGxvIGhlbGxvIiwgInVzZXJuYW1lIjogImJsYWggYmxhaCJ9="}
+
+validate(event=sample_event, schema=schemas.INPUT, envelope="powertools_json(powertools_base64(data))")
```

Comment:
I think we can do better as part of this refactoring - could you add a Lambda Handler and think of try/exception too (in case validation fails)?

I think we can have `sample_event` as a separate JSON too so we standardize any input as a separate JSON file

Similar comment on Parser as there's a great portion of customers who aren't full time devs

https://github.com/awslabs/aws-lambda-powertools-python/issues/1371#issuecomment-1203080410